schema/person-statement.json

```diff
@@ -38,10 +38,36 @@
       "codelist": "personType.csv",
       "openCodelist": false
     },
-    "missingInfoReason": {
-      "title": "Missing information reason(s)",
-      "description": "For PersonStatements with the type 'anonymousPerson' or 'unknownPerson' this field should contain an explanation of the reason that detailed information on the person is not provided. This may be a standard descriptive phrase from the source system, or a free-text justification.",
-      "type": "string",
+    "unspecifiedPersonDetails": {
+      "title": "Unspecified person details",
+      "description": "When this person has a 'personType' of 'anonymousPerson' or 'unknownPerson' information explaining this should be given using the reason and description.",
```

Comment:
This applies to the other description fields as well. Can we reformat from:

"When this person has a 'personType' of 'anonymousPerson' or 'unknownPerson' information explaining this should be given using the reason and description."

To:

"An explanation of why this person has a `personType` of 'anonymousPerson' or 'unknownPerson'. A `reason` MUST be specified."

i.e.

* back-ticks around properties
* single quotes around codes from codelists
* mentions of required child properties
* the initial part of the description being publisher/consumer-neutral: simply stating what is/should be in the field.